distribution.md

```diff
@@ -69,3 +69,6 @@ Each stack ID MUST only be present once.
 For a given stack, the `mixins` list MUST enumerate mixins such that no included buildpacks are missing a mixin for the stack.
 
 Fewer stack entries as well as additional mixins for a stack entry MAY be specified.
+
+### Builders
+Buildpacks may be distributed using builders, as described in the [builder extension spec](extensions/builder.md).
```

Comment:
hmm, this kind of moves it to a first-class citizen in the spec if we call it out in the distribution spec doesn't it? I don't think we do that with other things like `project.toml`.

Reply:
I wasn't sure if it did, because it just mentions it as an option. I'm happy to take it out, and we can debate adding it in separately.

Reply:
@jkutner should we go ahead and remove these lines?